test/rpc-server/add-module/add/response.golden

```diff
@@ -1 +1 @@
-{"jsonrpc":"2.0","id":1,"result":{"module":"NEW"}}
+{"jsonrpc":"2.0","id":1,"result":{"module":"m8e723e34b156f187e7cce8658dbb513244788a1ab81c1f496b909b1a2d1cc0c3"}}
```

Comment:
You need to add the `name-as-id` option to the add-module request, too so you can use the name `NEW` in the next request